packages/amazon-bedrock/src/bedrock-api-types.ts

```diff
@@ -35,13 +42,15 @@ export interface BedrockToolInputSchema {
   json: Record<string, unknown>;
 }
 
-export interface BedrockTool {
-  toolSpec: {
-    name: string;
-    description?: string;
-    inputSchema: { json: JSONObject };
-  };
-}
+export type BedrockTool =
```

Comment:
This is semantically weird. Would it be cleaner to do the `|` where `BedrockTool` is used instead?

Reply:
Or introduce a separate type?